internal/locale/locales/en-us.yaml

```diff
@@ -1896,6 +1896,8 @@ arg_state_use_namespace_description:
   other: The org/project namespace of the project that you wish to use, or just the project name if previously used
 err_use_commit_id_mismatch:
   other: "Cannot switch to the given commit ID. Please use [ACTIONABLE]`state checkout`[/RESET] to check out a specific commit ID and then try again."
+err_use_default_project_does_not_exist:
```

Comment:
Is it possible for this error message to include the namespace of the default project we were expecting?